rllib/algorithms/impala/impala.py

```diff
@@ -422,16 +425,6 @@ def validate(self) -> None:
             self.vtrace_clip_pg_rho_threshold
         )
 
-    @override(AlgorithmConfig)
```

Comment:
Not needed anymore with this PR. grad clipping has been universally moved into `Learner.postprocess_gradients()`.